python/ray/serve/tests/test_api.py

```diff
@@ -682,7 +682,8 @@ def f():
         client.create_endpoint("endpoint", backend=2)
     client.create_endpoint("endpoint", backend="backend")
 
-
+# This error is now just printed because it is run in the control loop, not in the API path.
+@pytest.mark.skip()
```

Comment:
@edoakes how should this be handled?

Reply:
Good question. We should remove this error and instead just print a warning if it currently isn't feasible and say this is fine if your cluster is autoscaling

Reply:
sgtm